conf/bookkeeper.conf

```diff
@@ -277,6 +277,13 @@ useV2WireProtocol=true
 #
 # ensemblePlacementPolicy=org.apache.bookkeeper.client.RackawareEnsemblePlacementPolicy
 
+# The DNS resolver class used for resolving network locations for bookies. The setting is used
```

Comment:
I wonder if it should be mentioned that this setting would be used for the bookie client used in bookies for autorecovery and compaction? 

It would also be worth mentioning that this setting should match the setting for bookie clients in broker. In broker.conf, ZkBookieRackAffinityMapping will be used when `bookkeeperClientRackawarePolicyEnabled=true`. This is mentioned in this comment: https://github.com/apache/pulsar/issues/151#issuecomment-268802489

Reply:
Good point. I'll resolve conflicts and update the comments.

Reply:
Done.

Reply:
btw. I'm not sure if bookie client is used in bookkeeper for other than autorecovery.